packages/layout/components/Logo/Logo.tsx

```diff
@@ -6,16 +6,20 @@ import { messages } from '../messages';
 
 import styles from './Logo.module.css';
 import LogoImage from './Logo.svg';
+import { useRouteLoaderData } from 'react-router';
+import type { RootLoader } from 'seven/app/root';
 
 const Logo = (props: SlotComponentProps) => {
   const { content } = props;
+  const rootData = useRouteLoaderData<RootLoader>('root');
+
   const intl: (id: string) => string = config.getUtility({
     name: 'translation',
     type: 'factory',
   }).method;
 
   const navRootPath = content['@components'].navroot?.navroot?.['@id'] || '/';
```

Comment:
Maybe we should make it a habit to get the content from the root loader instead of the props? Not necessarily now, but something that should be discussed imho